tests/test_rhymes.py

```diff
@@ -62,28 +62,30 @@ def test_get_stressed_endings():
     # mano
     # prisión
     lines = [
-        {'tokens': [{
-            'word': [
-                {'syllable': 'plá', 'is_stressed': True},
-                {'syllable': 'ta', 'is_stressed': False},
-                {'syllable': 'no', 'is_stressed': False}
-            ],
-            'stress_position': -3}
-        ]},
-        {'tokens': [{
-            'word': [
-                {'syllable': 'ma', 'is_stressed': True},
-                {'syllable': 'no', 'is_stressed': False}
-            ],
-            'stress_position': -2}
-        ]},
-        {'tokens': [{
-            'word': [
-                {'syllable': 'pri', 'is_stressed': False},
-                {'syllable': 'sión', 'is_stressed': True}
-            ],
-            'stress_position': -1}
-        ]}
+        {'tokens': [{'word': [{'syllable': 'plá', 'is_stressed': True},
+                              {'syllable': 'ta', 'is_stressed': False},
+                              {'syllable': 'no', 'is_stressed': False}],
+                     'stress_position': -3}],
+         'phonological_groups': [{'syllable': 'plá', 'is_stressed': True},
+                                 {'syllable': 'ta', 'is_stressed': False},
+                                 {'syllable': 'no', 'is_stressed': False}],
+         'rhythm': {'rhythmical_stress': '+-', 'type': 'pattern'},
+         'rhythmical_length': 2},
+        {'tokens': [{'word': [{'syllable': 'ma', 'is_stressed': True},
+                              {'syllable': 'no', 'is_stressed': False}],
+                     'stress_position': -2}],
+         'phonological_groups': [{'syllable': 'ma', 'is_stressed': True},
+                                 {'syllable': 'no', 'is_stressed': False}],
+         'rhythm': {'rhythmical_stress': '+-', 'type': 'pattern'},
+         'rhythmical_length': 2},
+        {'tokens': [{'word': [{'syllable': 'pri', 'is_stressed': False},
+                              {'syllable': 'sión', 'is_stressed': True}],
+                     'stress_position': -1}],
+         'phonological_groups': [{'syllable': 'pri', 'is_stressed': False},
+                                 {'syllable': 'sión',
+                                  'is_stressed': True}],
+         'rhythm': {'rhythmical_stress': '-+-', 'type': 'pattern'},
+         'rhythmical_length': 3}
```

Comment:
Shouldn't it be: `'rhythm': {'stress': '-+-', 'type': 'pattern', 'length': 3}`?

Reply:
Same as above